litmus-portal/frontend/src/components/Sections/Workflow/WorkflowCluster/index.tsx

```diff
@@ -62,7 +62,10 @@ const WorkflowCluster: React.FC<WorkflowClusterProps> = ({ gotoStep }) => {
 
   const handleClick = () => {
     getCluster({
-      variables: { project_id: userData.projectID, cluster_type: 'internal' },
+      variables: {
+        project_id: userData.selectedProjectID,
+        cluster_type: 'internal',
```

Comment:
What is meant for internal here?

Reply:
This is required as in the backend we are keeping the cluster type as internal, for connecting external clusters we will be offering this support 1.9 release onwards. (internal denotes the admin cluster or on which GQL server will be running, 'red cluster' as per the convention.) @chandankumar4